core/trino-main/src/main/java/io/trino/sql/planner/QueryPlanner.java

```diff
@@ -549,7 +551,9 @@ public PlanNode plan(Delete node)
                 assignmentsBuilder.putIdentity(symbol);
```

Comment:
Should we handle the redistribution case as well?

Reply:
Will add TODO

Reply:
`Kudu` and `Phoneix` does not utilize redistribution as well